app/views/users/index.html.erb

```diff
@@ -1,3 +1,7 @@
-<h1>Home Page</h1>
-
-<p>List of all users.</p>
\ No newline at end of file
+<div class="home-page">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+      <%= render partial: 'user_info', locals: { user: user } %>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly follow the given design, including position of `number of posts`, the border around each user

Your design
![image](https://user-images.githubusercontent.com/81932478/214692163-54e46bbb-4ab7-4b82-b858-eaa2ad31a27f.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/214692333-5a187f7f-84fd-4f3c-8404-c3e2ba597acd.png)